src/ui/public/vislib/lib/dispatch.js

```diff
@@ -226,16 +226,8 @@ define(function (require) {
      */
     Dispatch.prototype.highlightLegend = function (element) {
       var label = this.getAttribute('data-label');
-
       if (!label) return;
-
-      d3.select(element)
```

Comment:
Since the legend isn't part of the Vis object now, rather the visualize directive, we need to get at the parent to make this work. Css selectors and Jquery make this a bit more concise